docs/samples/README.md

```diff
@@ -72,6 +72,16 @@ Additionally KFServing also integrates with The [AI Explainability 360 (AIX360)]
 | Deploy Alibi Text Explainer| [Alibi Text Explainer](./explanation/alibi/moviesentiment) |
 | Deploy AIX360 Image Explainer| [AIX360 Image Explainer](./explanation/aix/mnist/README.md) |
 
+### Deploy InferenceService with Multiple Models(Alpha)
+Multi Model Serving allows deploying `TrainedModels` at scale without being bounded by the Kubernetes compute resources(CPU/GPU/Memory), 
+service/pod limits and reducing the TCO, see [Multi Model Serving](../MULTIMODELSERVING_GUIDE.md) for more details. 
+Multi Model Serving is supported for Triton, SKLearn/XGBoost as well as Custom KFServer.
+ 
+| Features  | Examples |
+| ------------- | ------------- |
+| Deploy multiple models with Triton Inference Server| [Multi Model Triton InferenceService](./mutlimodelserving/triton/README.md)  |
+| Deploy multiple models with SKLearn/XGBoost KFServer| [Multi Model SKLearn InferenceService](./mutlimodelserving/sklearn/README.md)  |
```

Comment:
Should be `./multimodelserving/` -- I believe there is a typo and the `l` and `t` need to be switched.